content/en/contribute/medic/product-development-process/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Product Development Process"
+linkTitle: "Product Development Process"
+weight: 1
+description: >
+  Explanation of the Continuous Discovery process used for product development at Medic
+---
+
+The Medic Product Team operates in a way that is most commonly referred to as [Continuous Discovery]({{% ref "continuous-discovery-overview" %}}). That process helps us to focus on the impact of our work and keep teammates well-aligned with the people who use the CHT.
+
+To allow teammates to truly learn from and empathize with those using the CHT, the Medic Product team has cross-functional subteams, called [Focused Working Groups]({{% ref "focused-groups" %}}), each aligned with different user types. Each group is focused on a specific set of users of the CHT and has an aligned roadmap. Roadmaps are made up of defined initiatives, which map to goals and strategies. There is a big emphasis on working towards outcomes (not only *outputs*). Product teammates work together, across disciplines, to understand opportunities for improvement, find solutions that will have an impact, and then build, deploy, and measure them.
+
+As part of responsible maintainers of a software system, software engineers also dedicate time to performing [Code Health]({{% ref "code-health" %}}) tasks. That ensures updates, upgrades, code enhancements, and small bugs are still handled while the team works on highly focused user-facing objectives.
+Each Focused Working Group has defined OKRs, which set their focus for a given quarter.
```

Comment:
Given the recent feedback on having more transparency into the product team, I think we should improve this line:
```suggestion
Each Focused Working Group has defined Objectives and Key Results (OKRs), which set their focus for a given quarter. These, in turn, are derived from the Product Team's yearly Objective Goals Strategies Measurements (OGSMs).  The [Schedule of Activities]({{% ref "contribute/medic/product-development-process/schedule-of-events" %}}) has more on the timing of all this through out any given year.
```

Then add OKR and OGSM to `contribute/medic/product-development-process/schedule-of-events`!

Reply:
Great one! Thank you for the addition